src/main/java/io/cdap/plugin/sfmc/source/MarketingCloudInputFormat.java

```diff
@@ -71,7 +71,8 @@
    * @param conf the plugin conf
    * @return Collection of MarketingCloudObjectInfo containing table and schema.
    */
-  private static List<MarketingCloudObjectInfo> fetchTableInfo(SourceQueryMode mode, MarketingCloudSourceConfig conf) {
+  protected static List<MarketingCloudObjectInfo> fetchTableInfo(SourceQueryMode mode,
```

Comment:
Why are we making this protected if we're just adding unit tests?

Reply:
Moved this method from protected to public